### ISSUE TITLE
Titles built as text tags should be labeled as headings for assistive technology

### ISSUE THREAD

**Issue by mpitblado** [gbif/hp-beaty-biodiversity-museum#80]
**Requirement**
Elements that visually appear as titles but are coded with a non-heading HTML Tag should include a "role" attribute that equals to "heading" or have their tags fixed. 

**2 Failed Elements**

- [ ] 1.
```html
<p class="tagLine">The Beaty Biodiversity Museum is Vancouver's natural history museum. <span data-ajax-url="https://api.gbif.org/v1/occurrence/search?institutionCode=BBM&amp;limit=0" class="ajax-is-loaded">641,448</span> objects across 3 collections... 
```

- [ ] 2.
```html
<span data-ajax-url="https://api.gbif.org/v1/occurrence/search?institutionCode=BBM&amp;limit=0" class="ajax-is-loaded">641,448</span> 
```
